lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -223,7 +239,15 @@ export class RedirectClient extends StandardInteractionClient {
                 this.browserStorage.cleanRequestByInteractionType(
                     InteractionType.Redirect
                 );
-                parentMeasurement.event.errorCode = "no_server_response";
+
+                // Do not instrument "no_server_response" if user clicked back button
+                if (getNavigationType() !== "back_forward") {
+                    parentMeasurement.event.errorCode = "no_server_response";
+                } else {
+                    this.logger.verbose(
```

Comment:
Should we have some other errorcode we log here specifically for clicking the back button?